blueflood-core/src/main/java/com/rackspacecloud/blueflood/io/serializers/NumericSerializer.java

```diff
@@ -535,7 +556,7 @@ private static AbstractRollupStat getStatFromRollup(byte statType, BasicRollup b
         public ByteBuffer toByteBuffer(Object o) {
             try {
                 byte type = typeOf(o);
-                byte[] buf = new byte[sizeOf(o, type)];
+                byte[] buf = new byte[sizeOf(o, type, VERSION_2_TIMER)];
```

Comment:
This is much cleaner.  I originally thought there was a problem doing it this way, but it seems much better.  I'll push some changes shortly.